pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "python-web3-wallet"
+version = "0.0.11"
+description = "Streamlit component that allows users to connect a wallet and send transactions with dynamic recipients and amounts"
+authors = ["Gnosis AI <ai@gnosis.io>"]
+license = "MIT"
```

Comment:
The one in LICENSE file is `LGPL-3.0 license`

Reply:
Thanks. Removed LICENSE to avoid inconsistency.